evm/src/main/java/org/hyperledger/besu/evm/fluent/SimpleBlockValues.java

```diff
@@ -15,7 +15,105 @@
  */
 package org.hyperledger.besu.evm.fluent;
 
+import org.hyperledger.besu.datatypes.Wei;
 import org.hyperledger.besu.evm.frame.BlockValues;
 
+import java.util.Optional;
+
+import org.apache.tuweni.bytes.Bytes;
+import org.apache.tuweni.bytes.Bytes32;
+
 /** A concrete BlockValues object that takes all the defaults */
-public class SimpleBlockValues implements BlockValues {}
+public class SimpleBlockValues implements BlockValues {
```

Comment:
non-blocking ofc,  but this class is not particularly fluent

Reply:
The EVMExecutor is fluent, and sets it via fluent .  This is then passed into the EVM.  Nested fluency seems pointless, and the block values hold the data rather than the executor

https://github.com/hyperledger/besu/pull/5930/files#diff-c691ed5f63ebd8d1a3908a282b65313fdea66411c51a77b2e811b5ba9c09e27fR873